src/collections/blog/2021-06-25-she-code-africa-contributhon-boot-camp/index.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "She Code Africa Contributhon Boot Camp Report"
+date: 2021-06-25 16:00:00 +0000
+author: Anita Charles
+thumbnail: ./sca-thumbnail.png
+category: Community
+tags:
+ - Community
+ - MeshMate
+published: true
+redirect_from: 
+ - /blog/community/she-code-africa-contributhon-boot-camp-report
+---
+
+import { Link } from "gatsby";
+
+import Sandy from "./sandySCA.jpg";
+import Abiola from "./abiolaSCA.jpg";
+import Cynthia from "./cynthiaSCA.jpg";
+import Elizabeth from "./elizabethSCA.jpg";
+import Blossom from "./blossomSCA.jpg";
+import Joy from "./joySCA.jpg";
+import Naimat from "./naimatSCA.jpg";
+
+import { BlogWrapper } from "../Blog.style.js";
+
+<BlogWrapper>
+  <p>
+  Layer5 participated as a suporting organization in the <a href="https://sites.google.com/shecodeafrica.org/contributhon" rel="nofollow">She Code Africa Contributhon</a> which started on April 1, 2021. It was indeed a pleasant and impactful experience for both the mentors and the mentees from She Code Africa.
+  </p>
+  <p>
+  The <a href="https://www.shecodeafrica.org" rel="nofollow">She Code Africa</a> hosts their Contributhon as an Africa-wide open source boot camp focused on building more women OSS contributors & creating more diversity in the African open source ecosystem by matching African women in technology with sponsors and mentors from Open Source organizations. During this boot camp, participants work with an open source organisation on a project for 4 weeks with the help of organisation mentors and get rewarded at the end of the boot camp upon successful project completion.
+  </p>
+
+  During the boot camp, Layer5 received a total of 8 mentees - all from Nigeria. These mentees were teamed up to work on the 3 project ideas selected by Layer5. Each project had an average of 2 members of the community who were delegated as mentors to oversee the contributions of the mentees. Often <Link to="/community/meshmates">Layer5 MeshMates</Link> volunteer to be mentors.
+
+  <h2>Projects</h2>
+  <p>Layer5 mentors prepared and organized a few projects in advance of the start of the boot camp, so that mentees might hit the ground running.</p>
+
+  <h3>Layer5 Community Handbook</h3>
+  
+  <p>
+    Layer5 Community Handbook is a curated Document with the ABC's of the community. In this project, mentees were expected to look for repetition pattern across existing docs thereby, creating a structured guide for interaction across all projects.
+  </p>
+  <b>Mentors:</b>
+  <ul>
+    <li><Link to="/community/members/ruth-ikegah">Ruth Ikegah</Link></li>
+    <li><Link  to="/community/members/anita-charles">Anita Ihuman</Link></li>
+  </ul>
+  <b>Mentee(s):</b>
+  <ul>
+    <li><Link to="community/members/anisat-akinbani">Anisat Akinbani</Link></li>
```

Comment:
This needs to be updated. It points to 404. @Anita-ihuman Can you create an issue to track and solve this?

Reply:
Oops. That is a bug that I introduced. It should be: <a href="https://twitter.com/AnisatAkinbani">Anisat Akinbani</a>